phpBB/adm/style/admin.css

```diff
@@ -2739,3 +2739,28 @@ fieldset.permissions .padding {
 	width: 1em;
 }
 /* stylelint-enable declaration-property-unit-whitelist */
+
+.acp-icon {
```

Comment:
Can you move this above `/* stylelint-enable declaration-property-unit-whitelist */` to prevent stylelint from flagging this?